src/services/DataStore.js

```diff
@@ -1,13 +1,20 @@
 import FirebaseClient from "./Firebase";
 
 const DB = FirebaseClient.firestore();
+
 if (process.env.NODE_ENV === "development") {
   DB.settings({
     host: "localhost:8080",
     ssl: false,
   });
+
   // Initialize collection - otherwise auth fails
-  DB.collection("users").doc("public_profiles").set({});
+  const profiles = DB.collection("users").doc("public_profiles");
+  profiles.get().then((doc) => {
+    if (!doc.exists) {
+      profiles.set({});
```

Comment:
Is this really the proper way to initialize document in firebase? It looks quite suspicious, but I haven't actually find any other way 👀

Reply:
that's the only way I know 🤷‍♂️